analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExceptionShouldNotBeThrownFromUnexpectedMethods.CSharp9.cs

```diff
@@ -1,27 +1,53 @@
 ﻿using System;
-using System.Runtime.CompilerServices;
+
+namespace Tests.CustomType
+{
+    public class ModuleInitializerAttribute : Attribute { }
+
+    public class Foo
+    {
+        [ModuleInitializer]
+        internal static void M1()
+        {
+            throw new Exception(); // Compliant
```

Comment:
From my point of view these comments are redundant if we just specify that the case is compliant without any other context info.

Can you please add a comment that the attribute is user defined? It took me a while to see this 😄